src/layer/__tests__/WmsLayer.ts

```diff
@@ -38,14 +41,17 @@ test('WmsLayer.getMapUrl returns an URL', () => {
   expect(imageUrl).not.toHaveQueryParams(['transparent']);
 });
 
-test('WmsLayer.getMap makes an appropriate request', () => {
+test('WmsLayer.getMap makes an appropriate request', async () => {
   const bbox = new BBox(CRS_EPSG4326, 19, 20, 20, 21);
   const layerId = 'PROBAV_S1_TOA_333M';
   const layer = new WmsLayer({
     baseUrl: 'https://proba-v-mep.esa.int/applications/geo-viewer/app/geoserver/ows',
     layerId,
   });
 
+  mockNetwork.reset();
+  mockNetwork.onAny().replyOnce(200, ''); // we don't care about response
```

Comment:
No, we really don't. ;-) 

This test checks only that the request to the service is correct - it doesn't check anything else. Which means that it doesn't matter what the response is, when the request is made, the test has already either passed or failed. 

Do you think the comment needs to be clarified?

Reply:
Talked live, I agree, this format checking doesn't belong here so we really don't care about the response.

Maybe an explanation of why we don't care about the response would be beneficial. I was thinking something like this would do (copied from the change below and modified):
`// we don't care about the response, we will just inspect the request params`

Reply:
Thank you, fixed!

Reply:
Perfect! Resolving.